iot-e2e-tests/edge-e2e/pom.xml

```diff
@@ -21,22 +21,25 @@
             <artifactId>junit</artifactId>
             <version>4.12</version>
         </dependency>
-        <dependency>
-            <groupId>com.google.guava</groupId>
-            <artifactId>guava</artifactId>
-            <version>28.2-jre</version>
-        </dependency>
```

Comment:
Why are you removing this dependency?

Reply:
weird, I didn't do this change explicitly. Taking a look.